noisyatom/blogs/views.py

```diff
@@ -0,0 +1,133 @@
+from urllib.parse import quote_plus
+
+from django.contrib import messages
+from django.contrib.contenttypes.models import ContentType
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.db.models import Q
+from django.http import HttpResponse, HttpResponseRedirect, Http404
+from django.shortcuts import render, get_object_or_404, redirect
+from django.utils import timezone
+
+from .forms import PostForm
+from .models import Post
+
+
+def create_post(request):
+	template_name = 'create.html'
+
+	if not request.user.is_staff or not request.user.is_superuser:
+		raise Http404
+
+	form = PostForm(request.POST or None, request.FILES or None)
+	if form.is_valid():
+		instance = form.save(commit=False) 
+		instance.user = request.user
+		instance.save()
+		messages.success(request, 'The page successfully created')
+		return HttpResponseRedirect(instance.get_absolute_url())
+	else:
+		messages.success(request, 'The page not created')
```

Comment:
Can you change to 'The page is not created' or 'This page has not been created' or 'This page failed to create'. 
Makes more sense in English... :-)

Reply:
done